From d2d123c640e2cb18e57d13bb7631a331c1018d8b Mon Sep 17 00:00:00 2001
From: Daniel Lo Nigro <daniel@dan.cx>
Date: Fri, 4 Dec 2015 18:45:38 -0800
Subject: [PATCH 100/214] Remove reference to 0.0.0.0

`0.0.0.0` doesn't really make much sense, and in fact shouldn't even work at all. It's pretty strange that it does work.
---
 README.md | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/README.md b/README.md
index 40f22e6..1a34f44 100644
--- a/README.md
+++ b/README.md
@@ -51,7 +51,7 @@ Edit `/_posts/2014-3-3-Hello-World.md` to publish your first blog post. This [Ma
 1. Install Jekyll and plug-ins in one fell swoop. `gem install github-pages` This mirrors the plug-ins used by GitHub Pages on your local machine including Jekyll, Sass, etc.
 2. Clone down your fork `git clone git@github.com:yourusername/yourusername.github.io.git`
 3. Serve the site and watch for markup/sass changes `jekyll serve`
-4. View your website at http://0.0.0.0:4000
+4. View your website at http://127.0.0.1:4000/
 5. Commit any changes and push everything to the master branch of your GitHub user repository. GitHub Pages will then rebuild and serve your website.
 
 ## Moar!
@@ -115,4 +115,4 @@ Issues and Pull Requests are greatly appreciated. If you've never contributed to
 
 You can start by [opening an issue](https://github.com/barryclark/jekyll-now/issues/new) describing the problem that you're looking to resolve and we'll go from there.
 
-I want to keep Jekyll Now as minimal as possible. Every line of code should be one that's useful to 90% of the people using it. Please bear that in mind when submitting feature requests. If it's not something that most people will use, it probably won't get merged. :guardsman:
\ No newline at end of file
+I want to keep Jekyll Now as minimal as possible. Every line of code should be one that's useful to 90% of the people using it. Please bear that in mind when submitting feature requests. If it's not something that most people will use, it probably won't get merged. :guardsman:
-- 
2.17.1

